kodilla-testing/src/main/java/com/kodilla/testing/forum/statistics/ForumStatistics.java

```diff
@@ -0,0 +1,43 @@
+package com.kodilla.testing.forum.statistics;
+
+public class ForumStatistics {
+    private int userCount;
+    private int postCount;
+    private int commentCount;
+    private double avrPostOnUser;
+    private double avrCommentOnUser;
+    private double avrCommentOnPost;
+
+    public void calculateAdvStatistics(Statistics statistics) {
+        this.userCount = statistics.usersNames().size();
+        this.postCount = statistics.postsCount();
+        this.commentCount = statistics.commentsCount();
+        this.avrPostOnUser = this.userCount != 0 ? (double)this.postCount / this.userCount : 0;
```

Comment:
ja bym tego nie dawał w konstruktorze (tych obliczeń), ale w metodach. Wtedy mamy tylko jedno pole w klasie, a mianowicie `statistics`

Reply:
poprawiłem to w następnym Commicie